.asf.yaml

```diff
@@ -88,6 +88,9 @@ github:
         - RPM Install Test Apache Cloudberry
         - ic-good-opt-off
         - ic-expandshrink
+        - ic-singlenode_isolation
+        - ic-singlenode_isolation2
+        - ic-resgroup-v2
```

Comment:
hi ed,

I don't quite understand how you turned on cgroup-v2？

Because the image we run on github CI is opened with CGroup V1 (you can only choose one of the two in linux kernel). And I asked relevant colleagues before, we have to use the same CI image to running CI?

Reply:
> hi ed,
> 
> I don't quite understand how you turned on cgroup-v2？
> 
> Because the image we run on github CI is opened with CGroup V1 (you can only choose one of the two in linux kernel). And I asked relevant colleagues before, we have to use the same CI image to running CI?

@jiaqizho Let me explain the cgroups v2 setup in our current CI environment.

We've made significant changes to our CI infrastructure. Previously, we used GitHub self-hosted runners with manually created Rocky Linux 8 VM images in AWS, which required substantial infrastructure maintenance. Now, we're using GitHub-hosted custom containers running Rocky Linux 9, which simplifies our setup considerably.

Our container configurations live in https://github.com/apache/cloudberry-devops-release/tree/main/images/docker/cbdb, and they're built through workflows at https://github.com/apache/cloudberry-devops-release/actions. These containers come with all necessary build and runtime packages pre-installed.

Regarding cgroups specifically:
- In the old Rocky Linux 8 environment, we used cgroups v1, configured via https://github.com/apache/cloudberry/blob/main/concourse/scripts/ic_gpdb_resgroup.bash
- With Rocky Linux 9, we switched to cgroups v2 (the current standard), previously configured using https://github.com/apache/cloudberry/blob/main/concourse/scripts/ic_gpdb_resgroup_v2.bash

The current workflow enables cgroups v2 through two main steps:
1. Starting the container with specific cgroups v2 support options
2. Running a "Setup cgroups" step that configures cgroups v2 for testing (based on enhanced logic from the original ic_gpdb_resgroup_v2.bash script)

For testing, we're using the installcheck-resgroup-v2 configuration. Note that I had to update the resgroup_memory_limit_resgroup_v2.out test output file to account for differences between cgroups v1 and v2 environments.

If it would be helpful, I'd be happy to schedule a video call to walk through this setup in more detail.